index.html

```diff
@@ -245,7 +244,7 @@ <h1 class="project-name">neveragain.tech</h1>
   <li><a href="https://responsibledata.io/">Responsible Data Forum</a> (data ethics)
   <li><a href="https://datasociety.net/">Data & Society</a> (research on social impact of data)
   <li><a href="http://www.acm.org/about-acm/acm-code-of-ethics-and-professional-conduct">ACM Software Engineering Code of Ethics and Professional Practice</a> (and <a href="http://ethics.acm.org/code-of-ethics/code-2018/">revisions committee</a>)
-  <li><a href="https://ssd.eff.org/">EFF&rsquo;s Surveillance Self-Defense Guide</a> 
+  <li><a href="https://ssd.eff.org/">EFF&rsquo;s Surveillance Self-Defense Guide</a>
```

Comment:
Hey there, can you submit a separate PR for this? We would have a larger discussion about adding a resource, but you're welcome to sign.